pages/builds/build_meta_data.md.erb

```diff
@@ -1,25 +1,51 @@
-# Using Build Metadata
+# Using Build Meta-data
 
 In this guide we’ll walk through using the Buildkite Agent’s [meta-data command](/docs/agent/v2/cli-meta-data) to store and retrieve data between different steps in a build pipeline.
 
 <%= toc %>
 
 ## Setting data
 
-To set meta-data you can simply:
+You can set meta-data via the command line or in a script. Both of these methods use the `buildkite-agent` cli with the `meta-data` command. Meta-data can only be set prior to starting an agent, it cannot be added once an agent is running.
+
+To set meta-data in the meta-data store, use the `set` command with a key/value pair:
 
 ```bash
 buildkite-agent meta-data set "release-version" "1.1"
 ```
 
+This command results in the value "1.1" being associated with the key "release-version" in the meta-data store. 
+
+To set meta-data on an agent, use the `meta-data` flag on the `start` command:
```

Comment:
Given this page is on Build meta-data, I reckon this might just confuse folks. I'd possible have a note about this confusion rather than this line?

E.g Note: The agent also has the concept of "meta-data" (renamed to "tags" in v3) for targeting builds to specific agents (e.g `queue=default`). This shouldn't be confused with the scoped key/value meta-data that the `buildkite-agent meta-data` command refers to!

Reply:
I was thinking about just removing it entirely... I left it in because I assumed that people searching for either one of the meta-datas in v2 would come to the meta-data page, and I wanted all the info to be there. 
Saying that, I'm happy to remove it, or add in your note. What do you think?

Reply:
Also, there's already a note in there to that effect, just after the code example. Should we remove mention of the flag and the example, but leave in the note?

Reply:
Yup, I reckon remove, or just a note as above to act as a disambiguator for v2.